### DISCUSSION TITLE
Milvus 2.4+ / 2.5+ Features Availability

### DISCUSSION THREAD

**Discussion by xiaobingxia-at** [milvus-io/milvus#38121] [Q&A and General discussion] [answered]
Hello, I'm reading the Milvus 2.5 release notes, which is amazing.
I have a question regarding the "clustering compaction" and "lazy load", I always thought they are already available at Milvus 2.4+, since I see the document and configuration already available. Is that the case? Or they are only available at 2.5+.

Thanks

**Comment by yhmo** [comment, marked as answer]
Right,  "clustering compaction" and "lazy load" are both available in v2.4 and v2.5.

**Reply by xiaofan-luan** [reply]
clustering compation and lazy load might not be good enough to put into production yet.
